types/src/transaction/block_epilogue.rs

```diff
@@ -21,16 +24,37 @@ pub enum BlockEpiloguePayload {
         block_id: HashValue,
         block_end_info: BlockEndInfo,
     },
+    V1 {
+        block_id: HashValue,
+        block_end_info: BlockEndInfo,
+        fee_distribution: FeeDistribution,
```

Comment:
Who creates the block epilogue transaction? The validator who proposed the block?